tests/test_tests.py

```diff
@@ -61,10 +73,25 @@ class MyDict(dict):
 
         assert tmpl.render(mydict=MyDict(), complex=complex(1, 2)).split() == [
             'False', 'True', 'False', 'True', 'True', 'False',
+            'True', 'False', 'False', 'True', 'False', 'False',
+            'True', 'True', 'False', 'True', 'False', 'True',
             'True', 'True', 'True', 'True', 'False', 'True',
             'True', 'True', 'False', 'True', 'True', 'True', 'True'
         ]
 
+    def test_booleans(self, env):
```

Comment:
I know all the old tests use this concatenation style, but let's use `pytest.mark.parametrize` for new tests, it's easier to reason about.

Reply:
Agreed. Fixed.